test/integration/proposals_config.ts

```diff
@@ -1,58 +1,16 @@
 import { ProposalCategory, ProposalsConfigMap } from '@custom-types/types';
 
-// import fip_xx_proposal from '@proposals/description/fip_xx';
-import fip_98 from '@proposals/description/fip_98';
-import fip_92 from '@proposals/description/fip_92';
+import fip_99 from '@proposals/description/fip_99';
 
 const proposals: ProposalsConfigMap = {
-  /*
-    fip_xx: {
-      deploy: true, // deploy flag for whether to run deploy action during e2e tests or use mainnet state
-      totalValue: 0, // amount of ETH to send to DAO execution
-      proposal: fip_xx, // full proposal file, imported from '@proposals/description/fip_xx.ts'
-      proposalId: '',
-      affectedContractSignoff: [''],
-      deprecatedContractSignoff: [''],
-      category: ProposalCategory.DAO
-    }
-    */
-  fip_98: {
-    deploy: false,
+  fip_99: {
+    deploy: true,
     proposalId: null,
-    affectedContractSignoff: [
-      'fei',
-      'voltFeiSwapContract',
-      'feiDAOTimelock',
-      'collateralizationOracle',
-      'voltDepositWrapper',
-      'voltOracle',
-      'volt',
-      'turboFusePCVDeposit',
-      'pcvGuardian'
-    ],
+    affectedContractSignoff: [],
```

Comment:
missing `affectedContractSignoff`

Reply:
Gonna be honest I have no idea how to use this

Reply:
Updated I figured out how to use this